apps/system/js/app_window.js

```diff
@@ -864,13 +863,11 @@
         }
       }
 
-      if (this.constructor.SUB_MODULES) {
-        for (var propertyName in this.constructor.SUB_MODULES) {
-          var moduleName = this.constructor.SUB_MODULES[propertyName];
-          if (moduleName) {
-            this[propertyName] = BaseModule.instantiate(moduleName, this);
-            this[propertyName].start();
-          }
+      for (var propertyName in this.constructor.SUB_MODULES) {
+        var moduleName = this.constructor.SUB_MODULES[propertyName];
+        if (moduleName) {
+          this[propertyName] = BaseModule.instantiate(moduleName, this);
+          this[propertyName].start();
```

Comment:
It's a little weird to me that we are instantiating and starting sub_modules in the installSubComponents function (shouldn't this be handled when the app_window is created?), but I guess it's already been like this so it's not part of this bug.